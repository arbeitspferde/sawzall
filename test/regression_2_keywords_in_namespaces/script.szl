proto "db.proto";
proto "event.proto";
proto "event.proto";
proto "exp_arg_file.proto";
proto "groningen_config.proto";
proto "params.proto";
