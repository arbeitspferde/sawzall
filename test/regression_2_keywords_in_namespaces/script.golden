type org.arbeitspferde.groningen.proto_.GroningenParams = parsedmessage {type PipelineSynchMode = int, static NONE: int = 0, static BASIC_SEMAPHORE: int = 1, static ITERATION_FINALIZATION_ONLY: int = 2, static PipelineSynchMode_names: map [enum_value: int] of enum_name: string = {2: "ITERATION_FINALIZATION_ONLY", 1: "BASIC_SEMAPHORE", 0: "NONE"}, static tagnames: map [string] of int = {"ErrorCode": 0, "input_log_name": 1, "duration": 2, "restart": 3, "latency_weight": 5, "throughput_weight": 6, "memory_weight": 8, "num_crossovers": 9, "mutation_prob": 10, "stagnant_gens": 11, "elite_count": 12, "pause_time_latency_score_percentile": 13, "number_of_executor_threads": 14, "pipeline_sync_type": 15, "health_check_timeout": 16, "health_check_poll_interval": 17, "health_check_server_type": 18, "metric_query_deadline_secs": 19, "metric_query_channel_shutdown_delay_secs": 20, "metric_query_retry_maximum_count": 21, "metric_query_retry_delay_secs": 22, "executor_wait_for_one_subject_restart_ms": 23, "executor_sleep_btwn_polling_metrics_ms": 24, "extractor_number_of_extractor_threads": 25, "maximum_inflight_subject_restart_count": 26, "subject_manipulation_deadline_ms": 27, "checkpoint_file": 28, "default_in_memory_cache_ttl": 29, "max_subjects_in_flight_fraction": 30, "subject_group_number_of_shuffles": 31, "subjects_to_display": 32, "event_log_prefix": 33, "event_log_rotate_size_bytes": 34, "event_log_flush_interval_secs": 35, "additional_exec_paths": 36, "requested_pipeline_id": 37}, input_log_name: string @ 1 : string, duration: int = 1440 @ 2 : int32, restart: int = 4 @ 3 : int32, latency_weight: float = 1.0 @ 5 : double, throughput_weight: float = 1.0 @ 6 : double, memory_weight: float = 1.0 @ 8 : double, num_crossovers: int = 1 @ 9 : int32, mutation_prob: float = 0.02 @ 10 : double, stagnant_gens: int = 0 @ 11 : int32, elite_count: int = 1 @ 12 : int32, pause_time_latency_score_percentile: float = 99.0 @ 13 : double, number_of_executor_threads: int = 3 @ 14 : int32, pipeline_sync_type: int = 0 @ 15 : int32, health_check_timeout: int = 30000 @ 16 : int32, health_check_poll_interval: int = 5000 @ 17 : int32, health_check_server_type: string = "" @ 18 : string, metric_query_deadline_secs: int = 3 @ 19 : int32, metric_query_channel_shutdown_delay_secs: int = 3 @ 20 : int32, metric_query_retry_maximum_count: int = 3 @ 21 : int32, metric_query_retry_delay_secs: int = 2 @ 22 : int32, executor_wait_for_one_subject_restart_ms: int = 30000 @ 23 : int32, executor_sleep_btwn_polling_metrics_ms: int = 60000 @ 24 : int32, extractor_number_of_extractor_threads: int = 10 @ 25 : int32, maximum_inflight_subject_restart_count: int = 1 @ 26 : int32, subject_manipulation_deadline_ms: int = 30000 @ 27 : int32, checkpoint_file: string = "" @ 28 : string, default_in_memory_cache_ttl: int = 600000 @ 29 : int32, max_subjects_in_flight_fraction: int = 3 @ 30 : int32, subject_group_number_of_shuffles: int = 8 @ 31 : int32, subjects_to_display: int = 5 @ 32 : int32, event_log_prefix: string = "alloc/logs/tmp-groningen_events" @ 33 : string, event_log_rotate_size_bytes: int = 524288000 @ 34 : int32, event_log_flush_interval_secs: int = 60 @ 35 : int32, additional_exec_paths: string = "" @ 36 : string, requested_pipeline_id: string @ 37 : string};
type org.arbeitspferde.groningen.proto_.ProgramConfiguration = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "cluster": 1, "user": 16, "jvm_search_restriction": 18, "deprecated_message_a": 19, "param_block": 20, "number_of_subjects": 21, "subject_warmup_timeout": 22}, type JvmSearchSpace = {type GcMode = int, static USE_CONC_MARK_SWEEP: int = 0, static USE_PARALLEL: int = 1, static USE_PARALLEL_OLD: int = 2, static USE_SERIAL: int = 3, static GcMode_names: map [enum_value: int] of enum_name: string = {3: "USE_SERIAL", 2: "USE_PARALLEL_OLD", 1: "USE_PARALLEL", 0: "USE_CONC_MARK_SWEEP"}, type Int64Range = {value: int @ 1 : int64, floor: int @ 2 : int64, ceiling: int @ 3 : int64, step_size: int @ 4 : int64}, gc_mode: array of int @ 1 : int32, adaptive_size_decrement_scale_factor: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 2, cms_exp_avg_factor: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 3, cms_incremental_duty_cycle: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 4, cms_incremental_duty_cycle_min: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 5, cms_incremental_offset: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 6, cms_incremental_safety_factor: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 7, cms_initiating_occupancy_fraction: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 8, gc_time_ratio: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 9, max_gc_pause_millis: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 10, max_heap_free_ratio: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 11, max_new_size: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 12, max_perm_size: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 13, min_heap_free_ratio: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 14, new_ratio: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 15, new_size: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 16, parallel_gc_threads: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 17, survivor_ratio: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 18, tenured_generation_size_increment: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 19, heap_size: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 20, young_generation_size_increment: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 21, cms_incremental_mode: bool @ 22 : bool, cms_incremental_pacing: bool @ 23 : bool, use_cms_initiating_occupancy_only: bool @ 24 : bool, soft_ref_lru_policy_ms_per_mb: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace.Int64Range @ 25}, type DeprecatedMessageA = {deprecated_a: string @ 16 : string, deprecated_b: array of string @ 15 : string}, type ClusterConfig = {type SubjectGroupConfig = {type ExtendedSubjectConfig = {subject_index: int @ 16 : int32, jvm_parameters: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace @ 17}, subject_config: array of org.arbeitspferde.groningen.proto_.ProgramConfiguration.ClusterConfig.SubjectGroupConfig.ExtendedSubjectConfig @ 1, subject_group_name: string @ 16 : string, user: string @ 17 : string, exp_settings_files_dir: string @ 18 : string, number_of_subjects: int @ 19 : int32, subject_warmup_timeout: int @ 20 : int32, restart_command: array of string @ 21 : string, number_of_default_subjects: int = 0 @ 22 : int32}, subject_group: array of org.arbeitspferde.groningen.proto_.ProgramConfiguration.ClusterConfig.SubjectGroupConfig @ 1, cluster: string @ 16 : string, user: string @ 17 : string, number_of_subjects: int @ 19 : int32, subject_warmup_timeout: int @ 20 : int32, number_of_default_subjects: int = 0 @ 21 : int32}, cluster: array of org.arbeitspferde.groningen.proto_.ProgramConfiguration.ClusterConfig @ 1, user: string @ 16 : string, jvm_search_restriction: org.arbeitspferde.groningen.proto_.ProgramConfiguration.JvmSearchSpace @ 18, deprecated_message_a: org.arbeitspferde.groningen.proto_.ProgramConfiguration.DeprecatedMessageA @ 19, param_block: org.arbeitspferde.groningen.proto_.GroningenParams @ 20, number_of_subjects: int = 0 @ 21 : int32, subject_warmup_timeout: int = 300 @ 22 : int32};
type org.arbeitpferde.groningen.proto_.PipelineId = parsedmessage {id: string @ 1 : string};
type org.arbeitpferde.groningen.proto_.Experiment = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "id": 1, "subject_ids": 2}, id: uint @ 1 : uint64, subject_ids: array of uint @ 2 : uint64};
type org.arbeitpferde.groningen.proto_.CommandLineArgument = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "name": 1, "value": 2}, name: string @ 1 : string, value: string @ 2 : string};
type org.arbeitpferde.groningen.proto_.SubjectConfiguration = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "start_timestamp": 1, "end_timestamp": 2}, start_timestamp: uint @ 1 : uint64, end_timestamp: uint @ 2 : uint64};
type org.arbeitpferde.groningen.proto_.ICms = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "time": 1, "dc": 2}, time_: float @ 1 : double, dc: int @ 2 : int32};
type org.arbeitpferde.groningen.proto_.CmsInitialMark = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "tenured_max_size": 1, "triggered_at_occupancy_size": 2}, tenured_max_size: int @ 1 : int32, triggered_at_occupancy_size: int @ 2 : int32};
type org.arbeitpferde.groningen.proto_.CmsRemark = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "start_size": 1, "total_size": 2}, start_size: int @ 1 : int32, total_size: int @ 2 : int32};
type org.arbeitpferde.groningen.proto_.YgOccupancy = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "current_occupancy": 1, "total_size": 2, "timestamp": 3}, current_occupancy: int @ 1 : int32, total_size: int @ 2 : int32, timestamp: float @ 3 : double};
type org.arbeitpferde.groningen.proto_.StampedTime = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "time": 1, "timestamp": 2}, time_: float @ 1 : double, timestamp: float @ 2 : double};
type org.arbeitpferde.groningen.proto_.Usage = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "wall_time": 1, "cpu_time": 2, "system_time": 3}, wall_time: float @ 1 : double, cpu_time: float @ 2 : double, system_time: float @ 3 : double};
type org.arbeitpferde.groningen.proto_.CommandLine = parsedmessage {argument: array of org.arbeitpferde.groningen.proto_.CommandLineArgument @ 1};
type org.arbeitpferde.groningen.proto_.ConcurrentModeFailure = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "cms_gen_size": 1, "cms_end_size": 2, "cms_time": 3, "perm_start_size": 4, "perm_end_size": 5, "perm_size": 6, "perm_time": 7, "tenured_start_size": 8, "tenured_end_size": 9, "tenured_size": 10, "cms_start_size": 11, "i_cms": 12}, cms_gen_size: int @ 1 : int32, cms_end_size: int @ 2 : int32, cms_time: float @ 3 : double, perm_start_size: int @ 4 : int32, perm_end_size: int @ 5 : int32, perm_size: int @ 6 : int32, perm_time: float @ 7 : double, tenured_start_size: int @ 8 : int32, tenured_end_size: int @ 9 : int32, tenured_size: int @ 10 : int32, cms_start_size: int @ 11 : int32, i_cms: org.arbeitpferde.groningen.proto_.ICms @ 12};
type org.arbeitpferde.groningen.proto_.ClassUnloading = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "when": 1, "classes": 2}, when_: org.arbeitpferde.groningen.proto_.StampedTime @ 1, classes: array of string @ 2 : string};
type org.arbeitpferde.groningen.proto_.Cms = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "time": 1, "cms_abort_preclean_due_to_time": 2, "cms_concurrent_abortable_preclean": 3}, time_: float @ 1 : double, cms_abort_preclean_due_to_time: float @ 2 : double, cms_concurrent_abortable_preclean: org.arbeitpferde.groningen.proto_.Usage @ 3};
type org.arbeitpferde.groningen.proto_.ParNew = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "abort_time": 1, "ages": 2, "desired_survivor_size": 3, "max_threshold": 4, "new_threshold": 5, "par_new_time": 6, "promotion_failed": 7, "age_survivor_sizes": 8, "survivor_start_size": 9, "survivor_end_size": 10, "survivor_delta_size": 11, "age_total_survivor_sizes": 12, "system_times": 13}, abort_time: float @ 1 : double, ages: array of int @ 2 : int32, desired_survivor_size: int @ 3 : int32, max_threshold: int @ 4 : int32, new_threshold: int @ 5 : int32, par_new_time: float @ 6 : double, promotion_failed: bool @ 7 : bool, age_survivor_sizes: array of int @ 8 : int32, survivor_start_size: int @ 9 : int32, survivor_end_size: int @ 10 : int32, survivor_delta_size: int @ 11 : int32, age_total_survivor_sizes: array of int @ 12 : int32, system_times: org.arbeitpferde.groningen.proto_.Usage @ 13};
type org.arbeitpferde.groningen.proto_.Subject = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "id": 1, "command_line": 2, "configuration": 3, "cluster_name": 4, "subject_group_name": 5, "user_name": 6, "subject_group_index": 7, "is_default": 8}, id: uint @ 1 : uint64, command_line: org.arbeitpferde.groningen.proto_.CommandLine @ 2, configuration: org.arbeitpferde.groningen.proto_.SubjectConfiguration @ 3, cluster_name: string @ 4 : string, subject_group_name: string @ 5 : string, user_name: string @ 6 : string, subject_group_index: uint @ 7 : uint64, is_default: bool = false @ 8 : bool};
type org.arbeitpferde.groningen.proto_.Gc = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "subject_id": 1, "id": 2, "partial": 3, "full": 4, "start_size": 5, "end_size": 6, "delta_size": 7, "record_counter": 8, "relative_timestamp": 9, "i_cms": 10, "cms": 11, "cms_remark": 12, "cms_abort_preclean_due_to_time": 13, "weak_refs_processing": 14, "scrub_symbols_and_string_tables": 15, "rescan_parallel": 16, "yg_occupancy": 17, "cms_concurrent_preclean_start": 18, "cms_concurrent_preclean": 19, "cms_concurrent_mark_start": 20, "cms_concurrent_mark": 21, "cms_concurrent_reset_start": 22, "cms_concurrent_reset": 23, "cms_concurrent_sweep_start": 24, "cms_concurrent_sweep": 25, "cms_concurrent_abortable_preclean_start": 26, "cms_concurrent_abortable_preclean": 27, "cms_initial_mark": 28, "par_new": 29, "concurrent_mode_failure": 30, "class_unloading": 31, "system_times": 32}, subject_id: uint @ 1 : uint64, id: int @ 2 : int32, partial: bool @ 3 : bool, full: bool @ 4 : bool, start_size: int @ 5 : int32, end_size: int @ 6 : int32, delta_size: int @ 7 : int32, record_counter: int @ 8 : int32, relative_timestamp: float @ 9 : double, i_cms: org.arbeitpferde.groningen.proto_.ICms @ 10, cms: org.arbeitpferde.groningen.proto_.Cms @ 11, cms_remark: org.arbeitpferde.groningen.proto_.CmsRemark @ 12, cms_abort_preclean_due_to_time: float @ 13 : double, weak_refs_processing: org.arbeitpferde.groningen.proto_.StampedTime @ 14, scrub_symbols_and_string_tables: float @ 15 : double, rescan_parallel: org.arbeitpferde.groningen.proto_.StampedTime @ 16, yg_occupancy: org.arbeitpferde.groningen.proto_.YgOccupancy @ 17, cms_concurrent_preclean_start: bool @ 18 : bool, cms_concurrent_preclean: org.arbeitpferde.groningen.proto_.Usage @ 19, cms_concurrent_mark_start: bool @ 20 : bool, cms_concurrent_mark: org.arbeitpferde.groningen.proto_.Usage @ 21, cms_concurrent_reset_start: bool @ 22 : bool, cms_concurrent_reset: org.arbeitpferde.groningen.proto_.Usage @ 23, cms_concurrent_sweep_start: bool @ 24 : bool, cms_concurrent_sweep: org.arbeitpferde.groningen.proto_.Usage @ 25, cms_concurrent_abortable_preclean_start: bool @ 26 : bool, cms_concurrent_abortable_preclean: org.arbeitpferde.groningen.proto_.Usage @ 27, cms_initial_mark: org.arbeitpferde.groningen.proto_.CmsInitialMark @ 28, par_new: org.arbeitpferde.groningen.proto_.ParNew @ 29, concurrent_mode_failure: org.arbeitpferde.groningen.proto_.ConcurrentModeFailure @ 30, class_unloading: org.arbeitpferde.groningen.proto_.ClassUnloading @ 31, system_times: org.arbeitpferde.groningen.proto_.Usage @ 32};
type org.arbeitpferde.groningen.proto_.PipelineState = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "id": 1, "configuration": 2, "subjects": 3}, id: org.arbeitpferde.groningen.proto_.PipelineId @ 1, configuration: org.arbeitspferde.groningen.proto_.ProgramConfiguration @ 2, subjects: array of org.arbeitpferde.groningen.proto_.Subject @ 3};
type org.arbeitpferde.groningen.proto_.EvaluatedSubject = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "subject": 1, "fitness": 2, "end_timestamp": 3}, subject: org.arbeitpferde.groningen.proto_.Subject @ 1, fitness: float @ 2 : double, end_timestamp: uint @ 3 : uint64};
type org.arbeitpferde.groningen.proto_.PipelineHistoryState = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "id": 1, "configuration": 2, "evaluated_subjects": 3, "end_timestamp": 4, "experiment_id": 5}, id: org.arbeitpferde.groningen.proto_.PipelineId @ 1, configuration: org.arbeitspferde.groningen.proto_.ProgramConfiguration @ 2, evaluated_subjects: array of org.arbeitpferde.groningen.proto_.EvaluatedSubject @ 3, end_timestamp: uint @ 4 : uint64, experiment_id: uint @ 5 : uint64};
;
type org.arbeitspferde.groningen.proto_.JvmFlag = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "name": 1, "value": 2, "managed_by_groningen": 3}, name: string @ 1 : string, value: string @ 2 : string, managed_by_groningen: bool @ 3 : bool};
type org.arbeitspferde.groningen.proto_.FitnessScore = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "name": 1, "score": 2, "coefficient": 3}, name: string @ 1 : string, score: float @ 2 : float, coefficient: float @ 3 : float};
type org.arbeitspferde.groningen.proto_.EventEntry = parsedmessage {type Type = int, static START: int = 0, static UNEXPECTED_DEATH: int = 1, static EXPERIMENT_END: int = 2, static Type_names: map [enum_value: int] of enum_name: string = {2: "EXPERIMENT_END", 1: "UNEXPECTED_DEATH", 0: "START"}, static tagnames: map [string] of int = {"ErrorCode": 0, "experiment_id": 1, "time": 2, "groningen_user": 3, "groningen_serving_address": 4, "subject_user": 5, "subject_serving_address": 6, "type": 7, "jvm_flag": 8, "score": 9, "groningen_configuration": 10, "pause_event": 11, "groningen_start_time": 12}, type JvmFlag = {name: string @ 1 : string, value: string @ 2 : string, managed_by_groningen: bool @ 3 : bool}, type FitnessScore = {name: string @ 1 : string, score: float @ 2 : float, coefficient: float @ 3 : float}, type PauseEvent = {duration_in_seconds: float @ 1 : double}, experiment_id: uint @ 1 : uint64, time_: uint @ 2 : uint64, groningen_user: string @ 3 : string, groningen_serving_address: string @ 4 : string, subject_user: string @ 5 : string, subject_serving_address: string @ 6 : string, type_: int @ 7 : int32, jvm_flag: array of org.arbeitspferde.groningen.proto_.EventEntry.JvmFlag @ 8, score: array of org.arbeitspferde.groningen.proto_.EventEntry.FitnessScore @ 9, groningen_configuration: org.arbeitspferde.groningen.proto_.ProgramConfiguration @ 10, pause_event: array of org.arbeitspferde.groningen.proto_.EventEntry.PauseEvent @ 11, groningen_start_time: uint @ 12 : uint64};
type org.arbeitspferde.groningen.proto_.UniqueSubjectIdentifierRow = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "experiment_id": 1, "event_time": 2, "groningen_serving_address": 3, "subject_serving_address": 4}, experiment_id: uint @ 1 : uint64, event_time: uint @ 2 : uint64, groningen_serving_address: string @ 3 : string, subject_serving_address: string @ 4 : string};
type org.arbeitspferde.groningen.proto_.UniqueExperimentIdentifierRow = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "experiment_id": 1, "groningen_start_time": 2, "groningen_serving_address": 3}, experiment_id: uint @ 1 : uint64, groningen_start_time: uint @ 2 : uint64, groningen_serving_address: string @ 3 : string};
type org.arbeitspferde.groningen.proto_.PauseTimeEventSummaryStatisticsRow = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "minimum_duration": 1, "maximum_duration": 2, "mean_duration": 3, "median_duration": 4, "standard_deviation_of_durations": 5, "total_duration": 6, "event_count": 7}, minimum_duration: float @ 1 : float, maximum_duration: float @ 2 : float, mean_duration: float @ 3 : float, median_duration: float @ 4 : float, standard_deviation_of_durations: float @ 5 : float, total_duration: float @ 6 : float, event_count: uint @ 7 : uint64};
type org.arbeitspferde.groningen.proto_.FitnessRow = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "score": 1, "total_score": 2}, score: array of org.arbeitspferde.groningen.proto_.FitnessScore @ 1, total_score: float @ 2 : float};
type org.arbeitspferde.groningen.proto_.JvmSettingsRow = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "flag": 1, "complete_commandline": 2}, flag: array of org.arbeitspferde.groningen.proto_.JvmFlag @ 1, complete_commandline: string @ 2 : string};
type org.arbeitspferde.groningen.proto_.ExperimentJoinRow = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "fingerprint_of_unique_subject_identifier": 1, "fingerprint_of_unique_experiment_identifier": 2, "experiment_id": 3}, fingerprint_of_unique_subject_identifier: string @ 1 : string, fingerprint_of_unique_experiment_identifier: string @ 2 : string, experiment_id: uint @ 3 : uint64};
;
;
type org.arbeitspferde.groningen.proto_.ExperimentArgs = parsedmessage {static tagnames: map [string] of int = {"ErrorCode": 0, "args": 1, "master_serving_address": 2, "tuned_arg": 3, "pipeline_id": 4}, args: string @ 1 : string, master_serving_address: string @ 2 : string, tuned_arg: array of string @ 3 : string, pipeline_id: string @ 4 : string};
;
;
;
